graviti/portex/base.py

```diff
@@ -255,3 +256,31 @@ def copy(self: _T) -> _T:
             setattr(obj, name, param.copy(getattr(self, name)))
 
         return obj
+
+
+def read_yaml(path: PathLike) -> PortexType:
+    """Read a yaml file into Portex type.
+
+    Arguments:
+        path: The path of the yaml file.
+
+    Returns:
+        A Portex type instance created from the input yaml file.
+
+    """
+    with open(path, encoding="utf-8") as fp:
+        return PortexType.from_pyobj(yaml.load(fp, yaml.Loader))
```

Comment:
Why not use `from_yaml`?

Reply:
Then should we change the input type of `from_yaml`? Since `yaml.load` accepts not only `str`.

Reply:
but json.loads not accept `fp` only `str`, I think we should change from_yaml and from_json together?